src/applications/vaos/appointment-list/components/BackendAppointmentServiceAlert.unit.spec.jsx

```diff
@@ -229,6 +229,68 @@ describe('VAOS Backend Service Alert', () => {
     });
   });
 
+  it('should not display BackendAppointmentServiceAlert if there is no failure returned on the past appointments list, useFeSourceOfTruthVA=true', async () => {
```

Comment:
@jenniemc requested to add unit tests for both when the feature toggle is on and off [previously](https://github.com/department-of-veterans-affairs/vets-website/pull/35498/files#r2018036014). Given that there are quite a lot of tests that depends on whether the feature toggle is on/off (see all the places where I added `"type": "VA"`), I've elected to pick one test in each suite to duplicate to give us some coverage. 

I don't think it's necessary to duplicate each test since I expect that we'll complete the conversion for all source of truth in the next few sprints and release them in production. Adding all the duplicated tests, then removing them once fully released seems unproductive.